pymongo/aggregation.py

```diff
@@ -29,7 +29,7 @@
     from pymongo.collection import Collection
     from pymongo.command_cursor import CommandCursor
     from pymongo.database import Database
-    from pymongo.pool import SocketInfo
```

Comment:
Could you open a new ticket to do this SocketInfo->Connection renaming as it's own PR? It makes this POC difficult to review.